natgeo/mortar
# [AEM-6921] Tweaks to broadsheet grid

https://jira.natgeo.com/browse/AEM-6921

To test:

- Pull AEM-6921
- `npm test`
- `gulp serve`
- verify layout at various breakpoints
- bonus points for testing in AEM, by running `gulp build`, pointing AEM at local mortar and building platform-view with mvn
